### ISSUE TITLE
Cannot Disable Control Buttons

### ISSUE THREAD

**Issue by SSTPIERRE2** [leecade/react-native-swiper#530]
### Which OS ?

iOS

### Version
Which versions are you using:

- react-native-swiper v? ^1.5.10
- react-native v0.?.? 0.44.0

### Expected behaviour

Ability to use some prop to selectively disable either control button (specifically to match the scrollEnabled={false} use case).

### Actual behaviour

Cannot disable either control button by any means.

### How to reproduce it>

To help us, please fork this component, modify one example in examples folder to reproduce your issue and include link here.

```
<Swiper
        showsButtons
        nextButton={<TouchableOpacity style={{ zIndex: 1 }} disabled><Image source=
                             {nextButtonImage} /></TouchableOpacity>}
        prevButton={<Image source={prevButtonImage} style={{ alignItems: 'center' }} />}
        scrollEnabled={false}
        loop={false}
      >
        <ComponentWithDisabledButtonUntilSomeConditionIsMet />
        <Component 2 />
</Swiper>
```
### Steps to reproduce

1. Create a basic swiper and a couple basic slides to test
2. Initially disable scrolling for the swiper
3. Attempt to match the control buttons to this behavior by disabling the forward button

### My Use Case

Basically, I have a slide in a swiper where the user can only proceed to the next slide if they meet a certain condition, so I need to selectively disable the nextButton.

I can see in the source code that the reason the code above to reproduce doesn't work is because the custom element you pass in becomes wrapped in a touchable which you cannot modify to disable, so I'm happy to PR a quick new prop if you'd like :)

**Comment by SSTPIERRE2** [comment]
Please ignore the first reference. I mistakenly PR'd to my own forked repo!

**Comment by SSTPIERRE2** [comment]
Looking for some review, please and thank you! I'll add any documentation or anything else I may have missed, just let me know.

**Comment by SSTPIERRE2** [comment]
@leecade anybody out there? haha

**Comment by SSTPIERRE2** [comment]
@arribbar Hello there I noticed you're active on here, would you like to check out my small proposition?

**Comment by arribbar** [comment]
#531 merged now
